src/Detector.php

```diff
@@ -64,6 +64,21 @@ public function determineMimeType(\SplFileObject $file): string
         return 'text/plain; charset=utf-8';
     }
 
+    /**
+     * Read width amd height from a image file
+     *
+     * @param string $filePath
+     */
+    public function determineDimensions(string $filePath): array
+    {
+        if (!file_exists($filePath)) {
+            throw new \InvalidArgumentException('File path ' . $filePath . ' is not existed');
```

Comment:
Please document this in the DocBlock:

```
@throws \InvalidArgumentException Unreadable SplFileInfo
```

-----

Also, as there's no test for this exception, it looks like that might actually be blocking merging because the coverage is lower. Can you see if there's any way to pass in an `SplFileInfo` that isn't readable for testing purposes?

I suppose we could try creating a PHPUnit mock that returns false for `isReadable`, and then we can assert this function was thrown.